bananas_cli/cli.py

```diff
@@ -27,7 +27,7 @@ async def cli(ctx, api_url, tus_url, client_id):
     Every option can also be set via an environment variable prefixed with
     BANANAS_CLI_; for example:
 
-    BANANAS_CLI_API_URL="http://localhost:8000" python -m bananas_clie
+    BANANAS_CLI_API_URL="http://localhost:8000" python -m bananas_cli
```

Comment:
Here too, I don't mind this in the same PR, but in the same commit is annoying for future-us :)

Reply:
I'll do these kind of things seperately 👍